fetch-utils.js

```diff
@@ -10,19 +10,19 @@ export function getUser() {
 
 export async function getFamilies() {
     // fetch all families and their bunnies
-
+    const response = await client.from('loving_families').select(`*, fuzzy_bunnies(*)`);
     return checkError(response);
 }
```

Comment:
Awesome job Mary! You could add ".match({ 'fuzzy_bunnies.user_id':  client.auth.session().user.id });" to allow for specific users to update their own families. You would also need a user_id column on your loving_families table.